src/OrchardCore.Modules/OrchardCore.Facebook/Manifest.cs

```diff
@@ -0,0 +1,25 @@
+using OrchardCore.Modules.Manifest;
+using OrchardCore.Facebook;
+
+[assembly: Module(
+    Name = "Facebook",
+    Author = "The Orchard Team",
+    Website = "http://orchardproject.net",
+    Version = "2.0.0",
+    Category = "Facebook"
+)]
+
+[assembly: Feature(
+    Id = FacebookConstants.Features.Login,
+    Name = "Facebook Login",
+    Category = "Facebook",
+    Description = "Auhenticates users from facebook.",
+    Dependencies = new[] { FacebookConstants.Features.Core }
+)]
+
+[assembly: Feature(
```

Comment:
You don't need this feature. If you don't flag its services with [OrchardFeature] then they are assigned to the module's default feature.